apps/communications/contacts/js/views/form.js

```diff
@@ -369,6 +371,36 @@ contacts.Form = (function() {
     };
   };
 
+  function showForm(event) {
+    if(event.key === 'Control') {
```

Comment:
Actually proper key codes for softkeys (``'MozSoftLeft' && 'MozSoftRight'``) were integrated into 2.2r branch 20-oct-2015, so you shall use proper key codes in your solution, otherwise it'll be completely useless. To enable proper keyCode for Left Softkey in emulator you will need attached patch for '[B2G-root]/frameworks/base' repository.
[0001-softkeys-enable-SoftLeft-key-processing-in-qemu.txt](https://github.com/harman-red-square/gaia/files/38681/0001-softkeys-enable-SoftLeft-key-processing-in-qemu.txt)

Listening for proper keyCode at least will let test team verify your solution. 

And after integration of softkeys engine additional changes will definitely be required: at least addition of softkey bar with proper items (this task is out of this PR's scope, since engine isn't in place yet)